test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj

```diff
@@ -30,7 +30,9 @@
                         :host "0.0.0.0"}
    :status-proxy       {:proxy-target-url "https://0.0.0.0:9001/ssl-status"
                         :ssl-opts common-ssl-config}
-   :web-router-service {:puppetlabs.trapperkeeper.services.status.status-proxy-service/status-proxy-service "/status-proxy"}})
+   :web-router-service {:puppetlabs.trapperkeeper.services.status.status-proxy-service/status-proxy-service "/status-proxy"}
+   ; Tell TK to not log errors to the screen. For tests with intentionally broken services
```

Comment:
the `invalid-proxy-config-throws` test outputs TK error log messages to the screen when running `lein test` for the intentionally improperly configured services. I decided to mute them with a `logback.xml` file. Not sure if this is reasonable.